views/admin/tpl/transaction_list.tpl

```diff
@@ -160,7 +160,15 @@ window.onload = function ()
                         <a href="Javascript:top.oxid.admin.editThis('[{$listitem->wdoxidee_ordertransactions__oxid->value}]');">[{$payment->oxpayments__oxdesc->value}]</a>
                     </td>
                     <td class="[{$listclass}]">
-                        <a href="Javascript:top.oxid.admin.editThis('[{$listitem->wdoxidee_ordertransactions__oxid->value}]');">[{$listitem->wdoxidee_ordertransactions__state->value}]</a>
+                        <a href="Javascript:top.oxid.admin.editThis('[{$listitem->wdoxidee_ordertransactions__oxid->value}]');">[{$listitem->wdoxidee_ordertransactions__state->value}]
+                          [{if $listitem->isPoiPiaPaymentMethod()}]
+                            [{if $listitem->wdoxidee_ordertransactions__parenttransactionid->value}]
+                                (matched)
```

Comment:
Where is this matched coming from is it a translation key?

Reply:
"unmatched" and "matched" are not translated, because they are added next to transaction state (which was not translated either).

Reply:
Please clear this with the BA i think this will be translated, but the decision is by the BA.

Reply:
We would propose to handle this additional request in another user story as it is out of scope for this one (affects the whole transaction table):
https://jira.parkside.at/browse/WDPD-208